sample-tracking-status-overview-domain/src/main/groovy/life/qbic/business/samples/count/CountSamplesInput.groovy

```diff
@@ -0,0 +1,19 @@
+package life.qbic.business.samples.count
+
+/**
+ * <b>Input interface for the {@link CountSamples} feature</b>
+ *
+ * <p>Provides methods to trigger the counting of samples for one project</p>>
+ *
+ * @since 1.0.0
+ */
+interface CountSamplesInput {
+
+    /**
+     * This method calls the output interface with the number of samples in the project
+     * and the number of samples that have been received.
+     * In case of failure the output interface failure method is called.
+     * @since 1.0.0
+     */
+    void countSamples(String projectCode)
```

Comment:
```suggestion
    void countReceivedSamples(String projectCode)
```

Reply:
We count the received as well as the total number of samples to be able to display the the fraction. Shall I still change it?

Reply:
Since it is about the received samples I would say yes.